src/checkers/community.js

```diff
@@ -33,7 +38,22 @@ TODO This needs to be filled out!`)
       if (file.name === 'contributing' && opts.contributing) {
         fileContent = await fs.readFileSync(path.join(__dirname, '../../', opts.contributing)).toString('utf8')
       } else {
-        fileContent = await fs.readFileSync(path.join(__dirname, `../../fixtures/${file.filePath}`)).toString('utf8')
+        // Get the readme.
+        // TODO: should this be branch specific?
+        const {data: readme} = await github.get(`/repos/${github.repoName}/readme`)
+        const vf = vfile(Buffer.from(readme.content, 'base64').toString('utf8'))
+        let contributing
+
+        // Use the section from the readme, if there is one.
+        const processor = unified().use(parse).use(find('contribute', {includeHeading: true}, function (section) {
```

Comment:
Break this into it's own function and unit test it on two different fixtures, one with contribute, one without, and see if the returned contributing section matches what you'd expect.